packages/ui/src/components/Checkbox/index.tsx

```diff
@@ -300,7 +303,9 @@ export const Checkbox = forwardRef(
               </CheckboxIconContainer>
             </StyledIcon>
           ) : null}
-          {children ? <label htmlFor={computedName}>{children}</label> : null}
+          <Tooltip text={tooltip}>
+            {children ? <label htmlFor={computedName}>{children}</label> : null}
+          </Tooltip>
```

Comment:
Let's not show the tooltip if there are no children:
```suggestion          
          {children ? (
            <Tooltip text={tooltip}>
              <label htmlFor={computedName}>{children}</label>
            </Tooltip>
          ) : null}
```